src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/LocalDB.Unix.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+namespace System.Data.SqlClient.SNI
+{
+    internal class LocalDB
+    {
+        internal static string GetLocalDBConnectionString(string localDbInstance)
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
Could you consider doing this (I'm asking for all throw of PNSE to do it)
```c#
   throw new PlatformNotSupportedException(); // some comment explaining why PNSE here
```

reason: we've had cases where something really should be implemented, was throwing temporarily or without complete thinking through, and then we don't know later whether it was thought through or not. A comment inline like this makes it super easy to grep the tree for "PlatformNoSupportedException" and find any missing comments.